.travis.yml

```diff
@@ -33,7 +35,7 @@ matrix:
         - popd
         - go install -race ./...
       script:
-        - go test -race -cover -coverprofile=.coverprofile ./...
+        - POSTGRESKV_TEST='postgres://postgres@localhost/pointerdb?sslmode=disable' go test -race -cover -coverprofile=.coverprofile ./...
```

Comment:
Note you can use:
```
- env:
    - MODE=tests
    - STORJ_POSTGRES_TEST='....'
  ...
```

Reply:
Ok, that would be a lot cleaner. I'm finding travis's yml schema to be quite confusing.